config/plugins.js

```diff
@@ -63,6 +63,20 @@ module.exports = ({ env }) => ({
       customLabels: {
         name: "strapi-prometheus",
       },
-    }
-  }
-});
\ No newline at end of file
+    },
```

Comment:
Seems like there is a conflict with the email plugin https://github.com/dev-launchers/strapiv4/blob/main/config/plugins.js#L68-L84.